<div class='alert alert-warning'>

SciPy's interactive examples with Jupyterlite are experimental and may not always work as expected. Execution of cells containing imports may result in large downloads (up to 60MB of content for the first import from SciPy). Load times when importing from SciPy may take roughly 10-20 seconds. If you notice any problems, feel free to open an [issue](https://github.com/scipy/scipy/issues/new/choose).

</div>

Test the null hypothesis that a random sample was drawn from a normal
distribution (with unspecified mean and standard deviation).


In [ ]:
import numpy as np
from scipy.stats import anderson
rng = np.random.default_rng()
data = rng.random(size=35)
res = anderson(data)
res.statistic

0.8398018749744764

In [ ]:
res.critical_values

array([0.527, 0.6  , 0.719, 0.839, 0.998])

In [ ]:
res.significance_level

array([15. , 10. ,  5. ,  2.5,  1. ])

The value of the statistic (barely) exceeds the critical value associated
with a significance level of 2.5%, so the null hypothesis may be rejected
at a significance level of 2.5%, but not at a significance level of 1%.